cmd/security-bootstrapper/entrypoint-scripts/consul_wait_install.sh

```diff
@@ -70,7 +70,7 @@ echo "$(date) ENABLE_REGISTRY_ACL = ${ENABLE_REGISTRY_ACL}"
 
 if [ "${ENABLE_REGISTRY_ACL}" == "true" ]; then
   echo "$(date) Starting edgex-core-consul with ACL enabled ..."
-  docker-entrypoint.sh agent \
+  exec docker-entrypoint.sh agent \
```

Comment:
consul is running as a background process and `exec` doesn't make sense here

Reply:
exec should only be always the last one command as it replaces the current program in the current process, without forking a new process

Reply:
Yea, was playing with this and forgot to remove and didn't see the `&`. Removed. THX!